test/cachedInstall.js

```diff
@@ -0,0 +1,97 @@
+// Copyright IBM Corp. 2014,2015. All Rights Reserved.
+// Node module: strong-cached-install
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+var async = require('async');
+var debug = require('debug')('strong-cached-install');
+var exec = require('child_process').exec;
+var fs = require('fs-extra');
+var ncp = require('ncp');
+var path = require('path');
+
+/**
+ * Install dependencies of the package in `appDir` directory.
+ * @param {string} appDir Application's root directory.
+ * @param {string} cacheDir Directory where to keep cached files.
+ * @param {Array<string>=} depTypes Optiona list of dependency types to install,
+ * e.g. `['dependencies', 'devDependencies']`. Defaults to `['dependencies']`.
+ * @param {function(Error=)} cb The callback.
+ */
+module.exports = function install(appDir, cacheDir, depTypes, cb) {
```

Comment:
Cached install is our module. If there is any fix needed to make loopback-workspace tests work, then please send a pull request to https://www.npmjs.com/package/strong-cached-install

We shouldn't be keeping an out-of-sync copy of that module in loopback-workspace.